src/core/backbone_router/bbr_local.cpp

```diff
@@ -65,18 +65,18 @@ Local::Local(Instance &aInstance)
     // All Network Backbone Routers Multicast Address.
     mAllNetworkBackboneRouters.Clear();
 
-    mAllNetworkBackboneRouters.GetAddress().mFields.m8[0]  = 0xff; // Multicast
-    mAllNetworkBackboneRouters.GetAddress().mFields.m8[1]  = 0x32; // Flags = 3, Scope = 2
-    mAllNetworkBackboneRouters.GetAddress().mFields.m8[2]  = 0;    // Reserved
-    mAllNetworkBackboneRouters.GetAddress().mFields.m8[15] = 3;    // Group ID = 3
+    mAllNetworkBackboneRouters.mFields.m8[0]  = 0xff; // Multicast
+    mAllNetworkBackboneRouters.mFields.m8[1]  = 0x32; // Flags = 3, Scope = 2
+    mAllNetworkBackboneRouters.mFields.m8[2]  = 0;    // Reserved
```

Comment:
I think it's fine to keep it as is, but can also consider optimizing/removing the `mAllNetworkBackboneRouters.mFields.m8[2]  = 0;`  since `Clear()` should set all bytes to zero.
(same for other addresses).